test/cache_test.rb

```diff
@@ -146,6 +151,49 @@ class InheritedRoleSerializer < RoleSerializer
       @blog_serializer     = BlogSerializer.new(@blog)
     end
 
+    def test_expiring_of_cache_at_update_of_record
+      if ARModels::Author.respond_to?(:cache_versioning)
+        ARModels::Author.cache_versioning = true
```

Comment:
we probably want to just change this value in the context of the test.  

```ruby
    def test_expiring_of_cache_at_update_of_record
      if ARModels::Author.respond_to?(:cache_versioning)
        original_cache_versioning = ARModels::Author.cache_versioning
        ARModels::Author.cache_versioning = true
     else
       original_cache_versioning = :no_cache_versioning
     end
     .....
   ensure
     ARModels::Author.cache_versioning = original_cache_versioning unless original_cache_versioning == :no_cache_versioning
   end
```

Reply:
neat!